helm/cluster-vsphere/files/etc/ssh/sshd_config

```diff
@@ -1,16 +0,0 @@
-# DEPRECATED - remove once CP and workers are rendered with cluster chart
```

Comment:
I got rid of everything marked DEPRECATED but let's check if there's more old stuff